editor/src/core/workers/parser-printer/parser-printer.ts

```diff
@@ -910,7 +910,7 @@ function printCodeImpl(
               missingImports.map((i) => {
                 return TS.createImportSpecifier(
                   false,
-                  TS.createIdentifier(i.name),
+                  i.name === i.alias ? undefined : TS.createIdentifier(i.name),
```

Comment:
What's this?

Reply:
This fixes the issue where you end up with `import { X as X } from 'x'`, which it would appear Prettier previously collapsed.

Reply:
Riiiight ok thanks